src/dsp_tools/commands/excel2json/models/ontology.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class Property:
+    name: str
+    super: list[str]
+    object: str
+    subject: str | None
+    labels: LanguageDict
+    comments: LanguageDict | None
+    gui_element: str
+    gui_attributes: GuiAttributes | None
+
+    def get(self) -> dict[str, Any]:
```

Comment:
Not sure, but perhaps this could be renamed to `serialize`? Just a thought, I don't have a strong opinion.